breathecode/registry/views.py

```diff
@@ -1809,6 +1801,11 @@ def get(self, request, cluster_slug=None, academy_id=None):
         if like is not None and like != "undefined" and like != "":
             items = items.filter(Q(slug__icontains=slugify(like)) | Q(title__icontains=like))
 
+        if "lang" in request.GET:
+            param = request.GET.get("lang")
+            if param:
+                items = items.filter(Q(lang__iexact=param) | Q(lang="") | Q(lang__isnull=True))
```

Comment:
You should only get those filtered by the language given, the code will look like this: `lookup["lang"] = lang`

Reply:
Add a validation to avoid filtering by lang when lang is an empty string

Reply:
1. If lang is in request.GET and has a value, it is added to lookup, then items is filtered with lookup.

2. Filtering by lang when it is an empty string: The condition if lang: ensures that it is only added to lookup if it has a value.